src/kili/entrypoints/mutations/asset/__init__.py

```diff
@@ -262,11 +286,23 @@ def update_properties_in_assets(
                 "status_array",
                 "is_used_for_consensus_array",
                 "is_honeypot_array",
+                "page_resolutions_array",
             ]
         }
         properties_to_batch = process_update_properties_in_assets_parameters(parameters)
 
         def generate_variables(batch: Dict) -> Dict:
+            page_resolutions_array_batch = []
+            for page_resolution_array in batch["page_resolutions_array"]:
+                output_page_resolution_array = []
+                for page_resolution in page_resolution_array:
+                    output_page_resolution_array.append(
+                        page_resolution.as_dict()
+                        if isinstance(page_resolution, PageResolution)
+                        else page_resolution
+                    )
+                page_resolutions_array_batch.append(output_page_resolution_array)
+
```

Comment:
shouldn't we put this logic in `process_update_properties_in_assets_parameters`?

Reply:
yes you're right